tests/js/bson-tests.ts

```diff
@@ -0,0 +1,63 @@
+'use strict';
```

Comment:
Any reason this is a `ts` file instead of a `js`?
If these tests are expected to pass in [the React Native application](https://github.com/realm/realm-js/tree/master/tests/react-test-app), I don't think we can use a typescript source file.

But I might be wrong?

Reply:
No real reason. Converted to plain js.